app/templates/application.hbs

```diff
@@ -1,3 +1,22 @@
-{{new-osf-navbar loginAction=(action 'login') hideSearch=true}}
+<div class="donate-banner">
+    <div class="platter">Public goods infrastructure should be free to use,
+but it's not free to build.</div>
+    <a  href="https://cos.io/donate-to-cos/" class="button"> Donate </a>
+</div>
```

Comment:
add a banner for donate.

Reply:
the banner could stand to be less intense and more in keeping with OSF style (flat design like you mentioned might fix it) - it's distracting right now, but not in a super positive way